kolibri/core/assets/src/views/AttemptLogList.vue

```diff
@@ -5,9 +5,21 @@
       {{ $tr('answerHistoryLabel') }}
     </h3>
 
+    <KSelect
+      v-if="isMobile"
+      class="history-select"
+      :value="selected"
+      label=""
```

Comment:
since the label is blank, could possibly use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) and point it at the `h3` above

Reply:
[Select element must have an accessible name](https://dequeuniversity.com/rules/axe/4.2/select-name)